packages/app/src/components/pages/NominationPage/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { useContext, useEffect } from 'react';
+import { ActiveNominationContext } from '../../../utils/context/ActiveNominationContext';
+import { NominationsDataContext } from '../../../utils/context/NominationsContext';
+import NominationBanner from '../../nominationBanner/nominationBanner';
+import ApplicationStages from '../../applicationStages/ApplicationStages';
+
+const NominationsPage = ({
+  match: {
+    params: { id },
+  },
+}) => {
+  const [activeNomination, setActiveNomination] = useContext(
+    ActiveNominationContext
+  );
+  const [NominationsData, setNominationsData] = useContext(
+    NominationsDataContext
+  );
+
+  useEffect(() => {
+    if (NominationsData) {
+      NominationsData.filter((nomination) => {
+        if (nomination.id === id) {
+          return setActiveNomination(nomination);
```

Comment:
Nitpick: If you're just looking to run a function against all the members of an array, `forEach` seems preferable to `filter`, which has a meaningful return value. You're not actually filtering here, and in other contexts, having "side effects" inside a filter's callback is (IMO) an anti-pattern.